src/Core/Settings/GlobalSettings.cs

```diff
@@ -29,7 +29,7 @@ public class GlobalSettings
         public virtual ConnectionStringSettings Storage { get; set; } = new ConnectionStringSettings();
         public virtual ConnectionStringSettings Events { get; set; } = new ConnectionStringSettings();
         public virtual NotificationsSettings Notifications { get; set; } = new NotificationsSettings();
-        public virtual FileStorageSettings Attachment { get; set; } = new FileStorageSettings();
+        public virtual IFileStorageSettings Attachment { get; set; } = new FileStorageSettings();
```

Comment:
Yes. Because we have a default value on the property, the Binder sees an instance and tries to fill in that instance with reflection, rather than having to instantiate the property itself.